R/reportEmi.R

```diff
@@ -1134,7 +1153,12 @@ reportEmi <- function(gdx, output = NULL, regionSubsetList = NULL,
                  setNames(dimSums(vm_emiMacSector[, , "co2luc"], dim = 3) * GtC_2_MtCO2,
                           "Emi|CO2|+|Land-Use Change (Mt CO2/yr)"),
                  # negative emissions from (non-BECCS) CDR (DACCS, EW)
-                 setNames((vm_emiCdrTeDetail[, , "weathering"] + vm_emiCdrTeDetail[, , "dac"] * p_share_CCS) * GtC_2_MtCO2,
+                 setNames((
+                  vm_emiCdrTeDetail[, , "weathering"]
+                  + vm_emiCdrTeDetail[, , "dac"] * p_share_CCS
+                  + (vm_emiCdrTeDetail[, , "oae_ng"] + vm_emiCdrTeDetail[, , "oae_el"])
+                    * (1 - s33_OAE_chem_decomposition * (1 - p_share_CCS))
```

Comment:
also subtract part that is never captured due to capture rate <100%, not just part that is not stored

Reply:
that part should not end up in gross emissions, but should be subtracted here

Reply:
It is now deducted from vm_emiCdr